pyvista/utilities/reader.py

```diff
@@ -156,7 +156,7 @@ def get_reader(filename, force_ext=None):
     >>> reader  # doctest: +ELLIPSIS
     XMLPolyDataReader('.../Human.vtp')
     >>> mesh = reader.read()
-    >>> mesh # doctest: +ELLIPSIS
+    >>> mesh  # doctest: +ELLIPSIS
```

Comment:
This is not a blackdoc issue, only a pointer to something potentially redundant: in `pyproject.toml` we have `doctest_optionflags = 'NUMBER ELLIPSIS'` so I wonder if the four `+ELLIPSIS` directives we have in `reader.py` are reduntant. To be looked at later if anyone feels like it.